src/store/reducers/executeQuery.ts

```diff
@@ -201,6 +240,19 @@ export const executeQueryApi = api.injectEndpoints({
                     }
 
                     const data = parseQueryAPIExecuteResponse(response);
+
+                    const queryStats: QueryStats = {};
+                    if (data.stats) {
+                        const {DurationUs, Executions: [{FinishTimeMs}] = [{}]} = data.stats;
+                        queryStats.duration = DurationUs;
+                        queryStats.endTime = FinishTimeMs;
+                    } else {
+                        const now = Date.now();
+                        queryStats.duration = (now - timeStart) * 1000;
+                        queryStats.endTime = now;
```

Comment:
What do you think about saving duration as `ms`? `us` is not very common thing, it's become important not to forget all that conversions (`*1000` or `parseUsToMs`).

In both cases I suggest adding metric to field name: `durationMs` or `durationUs`

Reply:
I keep existing format from api. we could replace `us` to `ms` in all places, but usually I prefer to keep original data as is (if api returns us - it means there are cases where this precision is important)